kafka_service/src/main/java/com/example/kafka_service/config/KafkaTopicConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.example.kafka_service.config;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.config.TopicBuilder;
+
+@Configuration
+public class KafkaTopicConfig {
+
+    @Value("${kafka.topic.name}")
```

Comment:
From my view It's better, laconic and convinient to use 1 property class instead of large using annotation @Value, especially when u use it in one place, u can do it as bean

Reply:
I prefer this way